include/NovelRT/Input/KeyState.h

```diff
@@ -12,7 +12,7 @@ namespace NovelRT::Input {
    * A set of key states.
    */
   enum class KeyState : int32_t {
-    Idle = 0,
+    Idle = -1,
```

Comment:
Why is `Idle = -1`? What is the default key state?

Reply:
it clashed with one of the GLFW key states returning `0`, so default is currently `GLFW_RELEASE`.

Reply:
Maybe it should be reordered and commented to make it clear what the GLFW values are:
```
Idle = -1,
KeyUp = GLFW_RELEASE, // 0
...
```

Reply:
Okay I'll do that quickly then.